src/Kernel/Context.class.st

```diff
@@ -629,6 +629,14 @@ Context >> doPrimitive: primitiveIndex method: meth receiver: aReceiver args: ar
 			ifFalse: [ ^self send: (arguments at: 2) to: (arguments at: 1) with: (arguments at: 3) lookupIn: (arguments at: 4) ]
 		].
 
+	primitiveIndex = 88 ifTrue: ["Process>>suspend"
```

Comment:
This is another important change to make debugging safe for stepping over #suspend method (there are cases when suspend is executed over real active process which is an actual UI process of the debugger).